pkg/trace/roundtripper_test.go

```diff
@@ -509,55 +509,3 @@ func (suite) initHeaderRoundTripperState(t *testing.T) *roundtripperState {
 
 	return &roundtripperState{recorder, server}
 }
-
-func (suite) TestRoundripNoTracer(t *testing.T) {
```

Comment:
We need to completely remove this test because now we panic when there is no tracer in the config.